introduction.md

```diff
@@ -1,2 +1,4 @@
 Hello I am the rmarrine
 My favorite Code School path is JavaScript
+* Git
+* JavaScript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?